staging/src/k8s.io/kubectl/pkg/polymorphichelpers/rollout_status.go

```diff
@@ -102,7 +102,7 @@ func (s *DaemonSetStatusViewer) Status(obj runtime.Unstructured, revision int64)
 	}
 
 	if daemon.Spec.UpdateStrategy.Type != appsv1.RollingUpdateDaemonSetStrategyType {
-		return "", true, fmt.Errorf("rollout status is only available for %s strategy type", appsv1.RollingUpdateStatefulSetStrategyType)
```

Comment:
just a small improvement suggestion - output `daemon.Spec.UpdateStrategy.Type` as an additional piece of information for easier troubleshooting.

Anyway, it is clearly a copy/paste error that can confuse,

/lgtm